### ISSUE TITLE
train_dreambooth_lora_flux_kontext.py: error: the following arguments are required: --instance_prompt

### ISSUE THREAD

**Issue by huassaue** [huggingface/diffusers#11973]
### Describe the bug

train_dreambooth_lora_flux_kontext.py: error: the following arguments are required: --instance_prompt

### Reproduction

accelerate launch --main_process_port 29513 train_dreambooth_lora_flux_kontext.py \
  --pretrained_model_name_or_path=black-forest-labs/FLUX.1-Kontext-dev  \
  --output_dir="kontext-i2i" \
  --dataset_name="/root/paddlejob/workspace/env_run/output/wanghuake/DATA/merged_output" \
  --image_column="output" --cond_image_column="file_name" --caption_column="instruction" \
  --mixed_precision="bf16" \
  --resolution=1024 \
  --train_batch_size=1 \
  --guidance_scale=1 \
  --gradient_accumulation_steps=4 \
  --gradient_checkpointing \
  --optimizer="adamw" \
  --use_8bit_adam \
  --cache_latents \
  --learning_rate=1e-4 \
  --lr_scheduler="constant" \
  --lr_warmup_steps=200 \
  --max_train_steps=1000 \
  --rank=16 \
  --seed="0" \

### Logs

```shell

```

### System Info

train_dreambooth_lora_flux_kontext.py: error: the following arguments are required: --instance_prompt

**Comment by sussyba390** [comment]
You do have problem on Cuda.

**Comment by xmuyoucai** [comment]
same problem
